src/com/lftechnology/java/training/dipak/employeemanagement/controller/EmployeeController.java

```diff
@@ -0,0 +1,261 @@
+					LOGGER.info("Please enter the department::");
+					department = sc.nextLine();
+					LOGGER.info("Please enter the address::");
+					address = sc.nextLine();
+					LOGGER.info(
+						"Please enter whether the user is terminated::(true/false)");
+					isTerminated = sc.nextBoolean();
+					LOGGER.info(
+						"Please enter the user type::(ADMIN->1/USER->2)");
+					int role2 = sc.nextInt();
+					if (role2 == 1) {
+						role = UserType.ADMIN;
+					}
+					else if (role2 == 2) {
+						role = UserType.USER;
+					}
+					else {
+						throw new Exception("Invalid value of the User type");
+					}
+
+					break;
+				}
+				catch (Exception ex) {
+					LOGGER.log(Level.INFO, "Exception occurred:", ex);
+					LOGGER.info("Please re-enter the values.");
+				}
+
+			}
+
+			e.setAddress(address);
+			e.setDepartment(department);
+			e.setFullName(fullName);
+			e.setIsTerminated(isTerminated);
+			e.setUserName(userName);
+			e.setPassword(password);
+			e.setRole(role);
+
+			EmployeeService ed = ServiceFactory.getEmployeeService();
+			count = ed.addEmployee(e);
+
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception type::{0}", ex);
+		}
+		return count;
+	}
+
+	/**
+	 * <p>This method takes the name of the employee to be terminated and call
+	 * terminateEmployee method in service layer. This method returns the number
+	 * of employees terminated.</p>
+	 * 
+	 * @param e
+	 * @param sc
+	 * @return count
+	 */
+	public int terminateEmployee(Employee e, Scanner sc) {
+
+		String fullName = null;
+		int count = 0;
+		try {
+			LOGGER.info("Please enter the employee fullname::");
+			fullName = sc.nextLine();
+			e.setFullName(fullName);
+
+			EmployeeService ed = ServiceFactory.getEmployeeService();
+
+			ed.terminateEmployee(e);
+
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception type::{0}", ex);
+		}
+		return count;
+	}
+
+	/**
+	 * <p>This method takes the input for the filters and calls the viewEmployee
+	 * method. </p>
+	 * 
+	 * @param e
+	 * @param sc
+	 * @return
+	 */
+	public ResultSet viewEmployee(Employee e, Scanner sc) {
+
+		String fullName = null;
+		String department = null;
+		String address = null;
+		try {
+			LOGGER.info("Please enter the employee fullname::");
+			fullName = sc.nextLine();
+
+			LOGGER.info("Please enter the department::");
+			department = sc.nextLine();
+
+			LOGGER.info("Please enter the address::");
+			address = sc.nextLine();
+
+			e.setFullName(fullName);
+			e.setAddress(address);
+			e.setDepartment(department);
+
+			EmployeeService ed = ServiceFactory.getEmployeeService();
+
+			ed.viewEmployee(e);
+
+		}
+		catch (Exception ex) {
+			LOGGER.log(Level.INFO, "Exception type::{0}", ex);
+		}
+		return null;
+	}
+
+	// public int deleteEmployee(Employee e,Scanner sc) {
+	// String fullName=null;
+	// String department=null;
+	// String address=null;
+	// int count=0;
+	// try {
+	//
+	// LOGGER.info("Please enter the employee username::");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)